module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SavedSearchesTest.php

```diff
@@ -173,14 +173,14 @@ public function testSearchHistory(): void
         // different problems in a single test.
         $page = $this->performSearch('foo \ bar');
         $this->findAndAssertLink($page, 'Search History')->click();
-        // We should see our "foo \ bar" search in the history, but no saved
-        // searches because we are logged out:
+        // We should see our "foo \ bar" search in the history, but a login link
```

Comment:
I can't parse this sentence. Should `but` be `and`?

Reply:
Fixed.